spdmlib/src/common/session.rs

```diff
@@ -89,6 +90,7 @@ pub struct SpdmSessionRuntimeInfo {
     pub message_k: Option<HashCtx>,
     pub message_f: Option<HashCtx>,
     pub message_m: Option<HashCtx>,
+    pub message_l: Option<HashCtx>,
```

Comment:
please rename message_m to digest_context_l1l2.

please remove message_l.

please remove message_k and message_f, and use digest_context_th.